mat64/symmetric.go

```diff
@@ -112,8 +119,12 @@ func (s *SymDense) reuseAs(n int) {
 			Data:   use(s.mat.Data, n*n),
 			Uplo:   blas.Upper,
 		}
+		s.cap = n
 		return
 	}
+	if s.mat.N > s.cap {
```

Comment:
You are right. The set is small, but non-zero. I feel safer though that we catch any case where this has happened.